ACE/ace/CDR_Base.h

```diff
@@ -205,7 +205,12 @@ class ACE_Export ACE_CDR
   typedef ACE_UINT16 UShort;
   typedef ACE_INT32 Long;
   typedef ACE_UINT32 ULong;
-  typedef ACE_UINT64 ULongLong;
+
+#   if defined (ACE_HAS_UINT64_T)
```

Comment:
This doesn't seem a correct solution, when for MacOSX a different type for ACE_UINT64 has to be used then make use of the ACE_UINT64_TYPE macro in the config-macos*.h file(s) and not do this for all platforms in this file just for CDR_Base.h

Reply:
I removed this in a subsequent commit.